.github/workflows/releasability_checks.yml

```diff
@@ -23,6 +23,15 @@ on:
       commit-sha:
         description: "Commit SHA"
         required: true
+      releasabily-env:
+        description: "For development purposes, the environment to use (prod 06..., staging 30..., or dev 59...). Defaults to production."
+        required: false
+        default: "064493320159"
+        type: choice
+        options:
+          - "064493320159"
+          - "308147251410"
+          - "597611216173"
```

Comment:
It might work however I think an effort should be done regarding UX:
![image](https://github.com/user-attachments/assets/86ca8181-e56d-40fc-b469-9fa89cb87297)

I'm pretty sure no one knows by heart what these numbers means (dev, staging prod)

What do you think about displaying in the list: dev, staging, prod instead?

I could find an example of implementation here: https://stackoverflow.com/a/75101805  (~4LOC changes)